lib/rubocop/cop/style/trailing_body_on_method_definition.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for trailing code after the method definition.
+      #
+      # @example
+      #   # bad
+      #   def some_method; do_stuff
+      #   end
+      #
+      #   # good
+      #   def some_method
+      #     do_stuff
+      #   end
+      #
+      class TrailingBodyOnMethodDefinition < Cop
+        include AutocorrectAlignment
+
+        MSG = 'Method body goes below definition.'.freeze
```

Comment:
The message is normally expressed with an imperative verb that tells the user what to do or not to do. E.g.: "Put the method definition body on its own line."